src/ir/drop.cpp

```diff
@@ -62,8 +62,10 @@ Expression* getDroppedChildrenAndAppend(Expression* curr,
     return builder.makeSequence(builder.makeDrop(curr), last);
   }
 
+  auto childIter = ChildIterator(curr);
   std::vector<Expression*> contents;
-  for (auto* child : ChildIterator(curr)) {
+  contents.reserve(childIter.getNumChildren() + 1);
```

Comment:
I'm not sure this is worth it. It will force an allocation to happen in the case where no child ends up pushed, where no allocation would happen otherwise. And that might be common.

It also makes the code slightly less readable. So unless we have good data showing a benefit, I'd leave this part as it was.

Reply:
Good point! I revert this part